frontend-react/src/components/Content/Templates/IACardGridTemplate.tsx

```diff
@@ -15,27 +15,71 @@ const ArraySection = ({
 }: {
     dirs: ContentDirectory[];
     title?: string;
-}) => (
-    <section>
-        {title ? <h3>{title}</h3> : null}
-        <IACardList dirs={dirs} />
-    </section>
-);
-
-const MapSections = ({ cMap }: { cMap: ContentMap }) => {
-    const arraySections: JSX.Element[] = [];
-    cMap.forEach((value, key) =>
-        arraySections.push(
-            <ArraySection dirs={value!!} title={key} key={key} />
-        )
+}) => {
+    const dirsSorted = dirs.sort(function (a, b) {
+        if (a.title < b.title) {
+            return -1;
+        }
+        if (a.title > b.title) {
+            return 1;
+        }
+        return 0;
+    });
+    const order = dirs.map((d) => dirsSorted.indexOf(d) + 1);
+
+    return (
+        <section>
+            {title ? <h2>{title}</h2> : null}
+            <IACardList dirs={dirs} order={order} />
```

Comment:
Since we already sort the elements in the proper order with `dirsSorted`, it feels redundant to then extract said order with indices in `order` then have to pass it to `<IACardList />` I think it might be better to keep `<IACardList />` "stateless" in the sense that it doesn't need to "know" what order the elements are in, simply that it's rendering the data.

Reply:
When I tested locally, if you remove the entire `order` concept from `<IACardList />`, it seem order correctly since `dirsSorted` is already correctly in order and then we can avoid inline styles as well.